cmd/main.go

```diff
@@ -453,12 +453,21 @@ var appCmds = []cli.Command{
 	updateCmd,
 }
 
+func printMCVersion(c *cli.Context) {
+	fmt.Fprintf(c.App.Writer, "%s version %s (%s/%s)\n", c.App.Name, c.App.Version, runtime.GOOS, runtime.GOARCH)
+	fmt.Fprintf(c.App.Writer, "Copyright (c) 2015-%s MinIO, Inc.\n", CopyrightYear)
+	fmt.Fprintf(c.App.Writer, "Licence AGPLv3 <https://www.gnu.org/licenses/agpl-3.0.html>\n")
+}
```

Comment:
We should also add `runtime.Version()` and as well as the `commit-id` I think - similar to `minio -v`

Reply:
> We should also add runtime.Version() and as well as the commit-id I think - similar to minio -v

Done